privacy/index.md

```diff
@@ -4,150 +4,562 @@ title: Privacy Policy
+- [Article 8. California Online Privacy Protection Act, (CalOPPA) explained](#article-8-california-online-privacy-protection-act-caloppa-explained)
+- [Article 9. Fair Information Practices, explained](#article-9-fair-information-practices-explained)
+- [Article 10. Children's Online Privacy Protection Act (COPPA)](#article-10-childrens-online-privacy-protection-act-coppa)
+- [Article 11. Do not track](#article-11-do-not-track)
+- [Article 12. Mobile devices](#article-12-mobile-devices)
+- [Article 13. No logs policy](#article-13-no-logs-policy)
+- [Article 14. Cookies](#article-14-cookies)
+- [Article 15. Third party behavioural tracking](#article-15-third-party-behavioural-tracking)
+- [Article 16. Data request](#article-16-data-request)
+- [Article 17. Jurisdiction and Applicable Law](#article-17-jurisdiction-and-applicable-law)
+- [Article 18. Amendments to the Privacy Policy](#article-18-amendments-to-the-privacy-policy)
+- [Article 19. Contact](#article-19-contact)
+
+---------------------------------------
+
+**Article 1. Where we collect Data/PII from**
+---------------------------------------------
+
+At present, Safing ICS Technologies GmbH runs four (4) services, these are as follows, Website, Forum, SPN, and Portmaster update server: these
+services collect different levels of data, however we aim to collect as
+little as possible, and all data received by us is treated with the
+strictest of confidence with privacy always in mind, all data is held
+securely.
 
-## Why we collect data and what we do with it
+**Article 2. Personal Identifiable Information & Data we collect**
+------------------------------------------------------------------
 
-Information is key in decision making. We don’t want to shoot into the blind, so we want to grasp what is going on around us.
-When it comes to collecting data, we do this in the most privacy respecting way possible. We do everything we can in house and we inform you of all third-party services we use.
-We are explicit in the wording about what we collect, and we do not collect more than we need.
+Although the information collected is basic and may not be classed as
+PII, we treat all data with the same respect and as such have listed all
+the data, we collect from the services we provide.
 
-We do not collect any PII (personal identifiable information). We only have what you give us willingly and on purpose. For example, when you sign up on our forum or subscribe to Gate17.
+**Website:**
 
-## How we guide the process of data collection
+The data we collect from Website Visitors:
 
-Our goal is to protect user privacy! We are careful about what we collect, and how we collect it. As a general principle, if we find a way to collect less data and still get the little information we need, we will do it.
+Includes Domains: <https://safing.io>, [https://docs.safing.io](https://docs.safing.io/)
```

Comment:
Why is one linked, the other not? Is the a criteria?

Reply:
just an fyi, both generate a link ;)